deepmd/utils/batch_size.py

```diff
@@ -16,6 +16,11 @@
     OutOfMemoryError,
 )
 
+try:
+    import paddle
+except ModuleNotFoundError:
+    pass
```

Comment:
I prefer moving the import to line 243 (make it an inline import). We should not import any deep learning framework outside their directories.

Reply:
Thanks, now move it into `concate_result` function